src/app/ReadClient.cpp

```diff
@@ -693,7 +693,10 @@ CHIP_ERROR ReadClient::RefreshLivenessCheckTimer()
     System::Clock::Timeout timeout =
         System::Clock::Seconds16(mMaxIntervalCeilingSeconds) + mpExchangeCtx->GetSessionHandle()->GetAckTimeout();
     // EFR32/MBED/INFINION/K32W's chrono count return long unsinged, but other platform returns unsigned
-    ChipLogProgress(DataManagement, "Refresh LivenessCheckTime with %lu milliseconds", static_cast<long unsigned>(timeout.count()));
+    ChipLogProgress(DataManagement,
+                    "Refresh LivenessCheckTime for %lu milliseconds with subscription id = 0x%" PRIx64 ", Peer = F:%" PRIx8
```

Comment:
why F and N, why not FabricID and NodeID? Seems more clear

Reply:
I'm trying to establish a compact logging format if you will that we can use in plenty of the other places where we log similar bits of information. In doing so, was trying to reduce the wordiness.

Reply:
I've just gone with `Peer = 01:0000000000000002`. I think that should be fairly self explanatory.

Reply:
In my experience, having brevity helps when scanning logs for debugging, which I've done many times in the past with Weave, which had a similar system. That brevity helped when scanning logs for figuring out whats happening.